### ISSUE TITLE
English version missing

### ISSUE THREAD

**Issue by tomsiwik** [elje/challenge#2]
I intend to merge all semi-projects and utility projects into one (not awesome-elje as it's for documentation, projectmngmt and tutorials). So far the following projects i consider frozen, deprecated and utility:

- [Sayme](https://github.com/elje/challenge) - Frozen
- [challenge](https://github.com/elje/challenge) - Not English, so not sure. For interviews? Can it be outsourced into gist.github.com or wiki?
- [tile_proxy](https://github.com/elje/tile_proxy) - Utility
- [transfer_history](https://github.com/elje/transfer_history) - Deprecated
- [transfer](https://github.com/elje/transfer) - Deprecated
- [christmas-map_map](https://github.com/elje/christmas-map_map) - Why not branch into aviapark_map ?
- [christmas-map_website](https://github.com/elje/christmas-map_website) - Why not branch into aviapark_website ?

So in short, what about this project?

Original issue & discussion found here: https://github.com/elje/awesome_elje/issues/1

**Comment by diedsmiling** [comment]
It's a test task for new developers. I've asked Olga in what language to write it and se asked to do it in Russian. ¯\_(ツ)_/¯
